app/packages/components/src/components/ModalBase/ModalBase.tsx

```
@@ -0,0 +1,253 @@
+          justifyContent: primaryButton?.align || "center",
+          ...secondaryButton,
+        },
+      },
+    },
+  });
+
+  // State options for functionality based on user input
+
+  {
+    /* TAGGING FUNCTIONALITY */
+  }
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)